setup.cfg

```
@@ -0,0 +1,49 @@
+[metadata]
+name = ukrdict
+url = https://github.com/serhii73/ukrdict
+project_urls =
+    CI: Travis = https://travis-ci.com/serhii73/ukrdict
+    GitHub: repo = https://github.com/serhii73/ukrdict
+    GitHub: issues = https://github.com/serhii73/ukrdict/issues
+author = serhii73
+author_email = aserhii@protonmail.com
+description = Library for searching ukrainian words meaning.
```

You should spell Ukrainian starting with a capital U as with any other languages.
```suggestion
description = CLI tool for looking up Ukrainian words meaning.
```